Modules/CMakeLists.txt

```diff
@@ -82,3 +82,5 @@ add_subdirectory(Skeleton)
 add_subdirectory(Daq)
 #add_subdirectory(ITS)
 add_subdirectory(SkeletonDPL)
+add_subdirectory(MyModule)
+add_subdirectory(Barth)
```

Comment:
We probably don't want these two in here

Reply:
indeed, sorry, I messed up